src/engraving/libmscore/keysig.cpp

```diff
@@ -437,7 +445,10 @@ void KeySig::read(XmlReader& e)
                     }
                     ks.sym = id;
                 } else if (t == "pos") {
-                    ks.spos = e.readPoint();
+                    int idx = _sig.keySymbols().size();
+                    ks.xPos = e.doubleAttribute("xPos", e.doubleAttribute("x", idx));  //if xPos not there, use x, or accidetal index
+                    ks.line = e.intAttribute("line", static_cast<int>(e.doubleAttribute("y", 2) * 2)); //if line not there, use y, or middle line
```

Comment:
here too

Reply:
And maybe put these comments above the line they belong to, because this way the lines are too long.